index.html

```diff
@@ -52,7 +53,170 @@ <h1>I'm CAkE <br>
             </div>
         </section>
 
+        <section class="works">

```

Comment:
- [ ] Kindly use a proper margin and padding during styling
- Your Design
![alignwell](https://user-images.githubusercontent.com/81444338/176995894-0c346f98-0699-49fc-91b6-50cbcf069f4b.PNG)
 - Figma Design
![Capturealign](https://user-images.githubusercontent.com/81444338/176996050-391bc647-40cc-41d4-821d-68f147bd37ae.PNG)